### DISCUSSION TITLE
Clear filter form in table component

### DISCUSSION THREAD

**Discussion by wit3** [filamentphp/filament#988] [Help]
Hi, anyone can explain how clear filter setted in table component?

**Comment by danharrin** [comment]
Hey, you'd need `$this->getTableFiltersForm()->fill()`.

**Reply by wit3** [reply]
Thanks for reply, but how implement a button that trigger an action in popup filter?

The only way is to publish and override blade vendor file?

**Reply by danharrin** [reply]
I'd accept a PR if the button looked neat :)
